.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "external/hapi-fhir-jpaserver-starter"]
 	path = external/hapi-fhir-jpaserver-starter
 	url = https://github.com/hapifhir/hapi-fhir-jpaserver-starter.git
+[submodule "cqf-ruler"]
+    path = cqf-ruler
+    url = https://github.com/DBCG/cqf-ruler.git
```

Comment:
This should be removed. It creates a recursive depending on this repository.

Reply:
Yeah, the CI build was giving an error saying that was needed: https://app.travis-ci.com/github/DBCG/cqf-ruler/builds/255666890

Seems to be building now though